packages/react/src/elements/components/GlobalNav/GlobalNav.js

```diff
@@ -50,6 +51,10 @@ export default class GlobalNav extends Component {
     );
   }
 
+  showProfile() {
+    return !!this.props.topNav.profile;
```

Comment:
Ideally this would check for a `null` prop or an empty object. If there's a quick handy way to do this, I'm interested; I usually depend on lodash for these things.

Reply:
At this point in the class you can should be able to assume `this.props.topNav.profile` is either `undefined` or an object. PropTypes should flag anything else.

`return !!this.props.topNav.profile;` works and `return this.props.topNav.profile !== undefined;` reads a little nicer.